client/app/kubernetes/nodes/kubernetes-nodes.controller.js

```diff
@@ -101,6 +101,10 @@ angular.module('managerApp').controller('KubernetesNodesCtrl', class KubernetesN
       });
   }
 
+  static isProcessing(node) {
+    return node.status === 'INSTALLING' || node.status === 'DELETING' || node.status === 'UPDATING' || node.status === 'RESETTING';
```

Comment:
Could you use constants please ? :)

Reply:
Also you could use `includes` here 
```suggestion
    return PROCESSING_STATUS.includes(node.status)
```

Reply:
Both the changes are done :)